retriever/engines/sqlite.py

```diff
@@ -106,3 +109,33 @@ def get_connection(self):
         import sqlite3 as dbapi
         self.get_input()
         return dbapi.connect(self.opts["file"])
+
+
+    @staticmethod
+    def fetch_tables(dataset, file= os.path.join(DATA_DIR, 'sqlite.db'),
+                table_name=None):
+        """Return sqlite dataset as list of pandas dataframe."""
+        import sqlite3
+        
+        con = sqlite3.connect(file)
```

Comment:
This looks like hard coding.
We should use the script instance. We know the engine, so we can instantiate the right engine(sqlite). The engine has the connection methods that can provide us with the connection cursor. 

For now we should get the table names from the  table`s property in the script object. This will only work for `Json` datasets and will fail for `.py` datasets.

Later, we shall re design the `.py` datasets to be able to get the tables initialized like the `Json` datasets